src/api/app/controllers/webui/request_controller.rb

```diff
@@ -288,12 +288,7 @@ def set_superseded_request
   end
 
   def require_request
-    required_parameters :number
-    @bs_request = BsRequest.find_by_number(params[:number])
-    return if @bs_request
-
-    flash[:error] = "Can't find request #{params[:number]}"
-    redirect_back(fallback_location: root_url) && return
+    @bs_request = BsRequest.find_by_number!(params[:number])
```

Comment:
```suggestion
    @bs_request = BsRequest.find_by!(number: params[:number])
```

Reply:
Sure, why not. I don't have a preference here.